tests/Integration/inc/Engine/Admin/Beacon/Beacon/insertScript.php

```diff
@@ -26,6 +26,7 @@ public function tearDown() {
 		parent::tearDown();
 
 		set_current_screen( 'front' );
+		delete_transient( 'wp_rocket_customer_data' );
```

Comment:
Could this transient already be set before entering into this test class? 

If yes or maybe, then you want to set it up to save and then restore the original. How? Add this property to class:

```php
protected static $transients = [
    'wp_rocket_customer_data' => null,
];
```

The `TestCase` handles the original save and restore stuff for you.